xml/admin_monitoring_alerting.xml

```diff
@@ -110,41 +110,87 @@
 
 <sect1 xml:id="grafana">
   <title>&grafana;</title>
+  <sect2 xml:id="grafana-certs">
+    <title>&grafana; SSL/TLS certificates</title>
+    <para>
+      All traffic is encrypted through &grafana;. You can either supply your
+      own SSL certs or create self-signed one.</para>
+    <para>&grafana; uses the following variables:</para>
+    <itemizedlist>
+      <listitem>
+        <para>
+          <emphasis role="bold"><literal>ssl_cert</literal></emphasis>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <emphasis role="bold"><literal>ssl_key</literal></emphasis>
+        </para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The &dashboard; embeds the &grafana; dashboards via HTML <literal>iframe</literal>
+      elements. If &grafana; is configured without SSL/TLS support or if SSL
+      uses self-signed certificates, most browsers will block the embedding of
+      insecure content into a secured web page if the SSL support in the
+      dashboard has been enabled (which is the default configuration). If you
```

Comment:
This is structured as:
if (condition 1), if (condition 2), result, if (condition 3(rider)).  That is not good for readability or comprehensibility.